From a2467f5af78c158f537ced482866ba4a334887c8 Mon Sep 17 00:00:00 2001
From: Ken Zhu <guigenz@codeaurora.org>
Date: Thu, 23 Apr 2020 19:58:58 -0700
Subject: [PATCH] dnsmasq: add isatap to the bogusname list

blacklist the domain names "wpad" and "isatap" from automated DNS record assignment
as in the CERT (VU#598349) required

Change-Id: I84342c81269be035759122728290545fe6341b1e
Signed-off-by: Ken Zhu <guigenz@codeaurora.org>
Signed-off-by: Karthik T S <quic_kartikts@quicinc.com>
(cherry picked from commit 89f104f780ddc2a7d52b34bcfea2fd7e0d3992ab)
---
 package/network/services/dnsmasq/files/dhcpbogushostname.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/network/services/dnsmasq/files/dhcpbogushostname.conf b/package/network/services/dnsmasq/files/dhcpbogushostname.conf
index e83b6975d03c..01b481ac037e 100644
--- a/package/network/services/dnsmasq/files/dhcpbogushostname.conf
+++ b/package/network/services/dnsmasq/files/dhcpbogushostname.conf
@@ -6,3 +6,4 @@
 
 dhcp-name-match=set:dhcp_bogus_hostname,localhost
 dhcp-name-match=set:dhcp_bogus_hostname,wpad
+dhcp-name-match=set:dhcp_bogus_hostname,isatap
-- 
2.34.1

